api/nakadi-event-bus-api.yaml

```diff
@@ -392,6 +394,46 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /event-types/{name}/effective_schema:
```

Comment:
Yeah I don't really like the alternative with having another field for event type :(
But maybe we can ask API guild about that?